### PR TITLE
PR TEST 2

### PR DESCRIPTION
**Pull request by ones1kk**

### CODE REVIEW

File: pull-request-test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+test
```
Comment (ghost) [inline pull-request-test.txt:1]: hi

Comment (ones1kk) [reply pull-request-test.txt:1]: hello

Comment (worker-hub) [inline pull-request-test.txt:1]: hello world

Comment (ones1kk) [reply pull-request-test.txt:1]: hello origin

**Review by worker-hub — APPROVED** [review]

**Pull request merged**